config/v1/types_ingress.go

```diff
@@ -82,8 +82,52 @@ type IngressSpec struct {
 	// Note that if there are no RequiredHSTSPolicies, any HSTS Policy annotation on the route is valid.
 	// +optional
 	RequiredHSTSPolicies []RequiredHSTSPolicy `json:"requiredHSTSPolicies,omitempty"`
+
+	// platform holds desired information specific to the underlying
+	// infrastructure provider for the ingress.
+	Platform *IngressPlatformSpec `json:"platform,omitempty"`
```

Comment:
Just want to reword this slightly
```suggestion
	// platform holds configuration specific to the underlying
	// infrastructure provider for the ingress load balancers.
	// When omitted, this means the user has no opinion and the platform is left
	// to choose reasonable defaults. These defaults are subject to change over time.
	// +optional
	Platform *IngressPlatformSpec `json:"platform,omitempty"`
```